static/src/stylesheets/module/content-garnett/_live-blog.scss

```diff
@@ -388,6 +388,29 @@ $block-padding-left: gs-span(1) + 4;
     padding: 3px 0 1px 5px;
 }
 
+.pinned-icon-situ {
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    min-width: 32px;
```

Comment:
Could / should the `px` values be REM? I think that's how they are in the rest of Frontend.

Reply:
I was also doubtful about this. I noticed that in most places in frontend they are using `px` in scss but it seems when it's converted to css, the `px`s are also converted to `rem`. So in this case `32px` is rendered as `2rem`